### ISSUE TITLE
Reversing legs in table view

### ISSUE THREAD

**Issue by AndrewAkinson** [richsmith/sexytopo#21]
Hmm how do I set something as an enhancement?

Currently you can reverse a leg from the plot screens, although this is a good place, it is often easier to do it in the data screen, would it be possible to add this as an option, please?

Also instead of having to pick select mode, could a context menu not be brought up with a long click on move mode? This would make it far easier for the user.

**Comment by richsmith** [comment]
I've tried to encourage use of the sketch view as much as possible rather than manipulating the raw data. But it would probably be easy enough to add leg reversing in the table view.

I'm not sure how easy it is to add a long press on move mode. The select tool currently has one press for select station and two for context menu so it's not clear which one a long press would do (first one, then the other?)

**Comment by AndrewAkinson** [comment]
Although preference would be to use sketch view, at times it becomes really difficult due to the mass overlap, as I had in Wookey.

Yes long press is not that important, just nice to have, found it tedious having to choose select to get the functionality of a menu.

**Comment by richsmith** [comment]
I'd like to experiment with a few extra gestures. Some way of moving in draw mode and shaking the device for Disto reconnection (for example). Do you mean passage overlap? I was hoping that would be largely solved by the new survey linking but I haven't really used it in the wild yet.

**Comment by richsmith** [comment]
Long press now selects a station (no matter what you're doing) and enters "Select" mode, ready for another press to enter Context menu. Open to suggestions as to whether it would be better to go directly into context menu or whether or not Select mode should be automatically selected.

(BTW, they're easier to manage if separate issues are on separate, well, issues!)

**Comment by AndrewAkinson** [comment]
issues, yes in principle that is goid, but sometimes they become
intertwined, I'll try harded.

Mt feelibgvis direct to context, but can I play with an update first,
before full commitment?

Now yet again, two issues in a single 'issue'

Andrew

**Comment by richsmith** [comment]
Yep, I have submitted the new version. Should be winging it's way to you over Google's internet tubes as I type!

**Comment by richsmith** [comment]
Also, strictly agile development. We can change minds about how it works at any point :)

**Comment by richsmith** [comment]
Done for latest version. Unified menus across data / sketch view.
